controllers/taloscontrolplane_controller.go

```diff
@@ -273,6 +273,10 @@ func (r *TalosControlPlaneReconciler) Reconcile(ctx context.Context, req ctrl.Re
 			"Scaling down control plane to %d replicas (actual %d)",
 			desiredReplicas, numMachines)
 
+		if numMachines == 1 {
+			return res, fmt.Errorf("failed to scale down control plane: can't scale down to 0 nodes")
```

Comment:
should we set this is a condition user might see?

and I think this deserves a test, as this is :fearful:

Reply:
sure, let's do it

Reply:
do we want to return an _error_ here, as this will cause reconcilation to be called once again?

I don't think controller can do anything at this point